moveit_ros/planning/trajectory_execution_manager/include/moveit/trajectory_execution_manager/trajectory_execution_manager.h

```diff
@@ -288,6 +291,7 @@ class TrajectoryExecutionManager
   bool execution_duration_monitoring_;
   double allowed_execution_duration_scaling_;
   double allowed_goal_duration_margin_;
+  double allowed_start_deviation_;
```

Comment:
add comment here explaining units - for revolute joints this would be tolerance of radians amount correct?